typescript.example/examples/modules/single-import/export.ts

```diff
@@ -0,0 +1,2 @@
+filename:"export"
```

Comment:
This is an necessary evil as I was not able to find out how you can resolve filenames to path locations, to be able to uniquely retrieve file locations (and thus types) in a project. If there is a solution, we should use it over this hack